starknet_py/tests/e2e/docs/devnet_utils/test_l1_integration.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from starknet_py.hash.selector import get_selector_from_name
+
+
+@pytest.mark.skip(reason="Test require eth node running.")
+@pytest.mark.asyncio
+async def test_postman_load(devnet_client, l1_l2_contract, account):
+    # pylint: disable=import-outside-toplevel
+    # pylint: disable=unused-variable
+
+    eth_account_address = 1390849295786071768276380950238675083608645509734
+
+    # docs: start
+    from starknet_py.devnet_utils.devnet_client import DevnetClient
+
+    client = DevnetClient(node_url="http://127.0.0.1:5050")
+
+    # docs: end
+    client: DevnetClient = devnet_client
+    # docs: start
+    # Load the messaging contract on ETH network
```

Comment:
To be consistent with the rest of the comments. Also imperative mode suggests that it is about something that must be done in this place, rather than an explanation of the code below.
```suggestion
    # Deploying the messaging contract on ETH network
```